pkg/frontend/frontend.go

```diff
@@ -407,11 +421,11 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 
 func (f *frontend) setupRouter() *mux.Router {
 	r := mux.NewRouter()
-	r.Use(middleware.Log(f.env, f.auditLog, f.baseLog.WithField("component", "access")))
```

Comment:
I don't see this log field being added in the new changes, is its remove purposeful and why don't we want it anymore then?

Reply:
fixed